python/core/classes-i/creating-classes.md

```diff
@@ -118,4 +123,34 @@ new empl = Employee("John")
 
 To explain what methods are, let's take a look at an example.
 
-Think of a car as the object. The model, color or age of the car are the object's attributes. Whether the engine is on, or whether it's moving, are behaviors and would exist as methods.
\ No newline at end of file
+Think of a car as the object. The model, color or age of the car are the object's attributes. Whether the engine is on, or whether it's moving, are behaviors and would exist as methods.
+
+[2: Block Structure]
+A block structure in python is determined with indentation.
+
+Take this piece of code for instance:
+```python
+class Employee:
+    company = 'Enki'
+
+    def __init__(self):
+        print('Hello Enki')
+ 
+    def message(self):
+        print('Welcome to Enki')
+ 
+emp = Employee() # Created an Instance
+
+print('My company is:', emp.company)
+emp.message()
```

Comment:
for the other examples in this insight we used 2 spaces for indentation. let's do the same for this code block:
```suggestion
class Employee:
  company = 'Enki'

  def __init__(self):
    print('Hello Enki')
 
  def message(self):
    print('Welcome to Enki')
 
emp = Employee() # Created an Instance

print('My company is: ', emp.company)
emp.message()
```